corehq/apps/locations/static/locations/ko/location.js

```diff
@@ -0,0 +1,33 @@
+// for product and user per location selection
+$(function () {
+    $("#id_users-selected_ids").width(800).height(400).multiselect();
+    $("#id_products-selected_ids").width(800).height(400).multiselect();
+});
+
+var insert_new_user = function(user) {
+    $('#id_users-selected_ids').multiselect('addSelected', user.user_id, user.text);
```

Comment:
Haha, yeah - I think this comes from slowly building up the id across a few libraries